prediction/src/tests/test_endpoints.py

```diff
@@ -4,13 +4,13 @@
 
     Provides unit tests for the API endpoints.
 """
+from flask import url_for
 from functools import partial
 import json
-
 import pytest
-
-from flask import url_for
-from src.factory import create_app
+import sys
+sys.path.append('.')
+from src.factory import create_app  # NOQA
```

Comment:
Hm, it's strange for this test to begin failing with this change.

It's probably because the change adds `__init__.py`s in `prediction` and `interface`. Two things to try instead of changing `sys.path`:
 (1) Does doing a relative import work? (e.g. `from .factory`)
 (2) Does importing `prediction.src.factory` work?

Other than that, we can also look at the call to `pytest`. IIRC there are a couple ways to do that which change how imports work for the test cases.